core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/instancegroup/network/InstanceGroupNetworkV4ToInstanceGroupNetworkConverter.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.cloudbreak.converter.v4.stacks.instancegroup.network;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.instancegroup.network.InstanceGroupNetworkV4Request;
+import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+import com.sequenceiq.cloudbreak.domain.stack.instance.network.InstanceGroupNetwork;
+
+@Component
+public class InstanceGroupNetworkV4ToInstanceGroupNetworkConverter
+    extends AbstractConversionServiceAwareConverter<InstanceGroupNetworkV4Request, InstanceGroupNetwork> {
+
+    @Override
+    public InstanceGroupNetwork convert(InstanceGroupNetworkV4Request source) {
+        InstanceGroupNetwork network = new InstanceGroupNetwork();
+        if (source.getAws() != null) {
```

Comment:
why don't you use the `ProviderParameterCalculator providerParameterCalculator;`?